telegram_inbound_handlers.py

```
@@ -0,0 +1,446 @@
+                    run.run_id, run.status, intent,
+                )
+            except Exception as exc:  # noqa: BLE001
+                log.exception("telegram_inbound: orchestrator.execute failed: %s", exc)
+
+        try:
+            import asyncio as _aio
+            from telegram_bot import _bg_tasks
+            _task = _aio.create_task(_run_in_background())
+            _bg_tasks.add(_task)
+            _task.add_done_callback(_bg_tasks.discard)
+        except Exception as exc:  # pragma: no cover
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)